src/features/InteractWithContracts/components/Interact.tsx

```diff
@@ -154,7 +155,7 @@ const FormSchema = object().shape({
   address: object({
     value: string().test(
       'check-eth-address',
-      translateRaw('TO_FIELD_ERROR'),
+      translateRaw('TO_FIELD_ERROR', { $network: DEFAULT_NETWORK }),
```

Comment:
I provided `network` to avoid displaying it as `Enter a valid $network address, ENS name, or blockchain domain`

Reply:
Couldn't you move the schema inside the component so it can access the `network` prop and use that instead?